examples/react-alt/js/actions/todoActions.js

```diff
@@ -0,0 +1,46 @@
+/*jshint quotmark:false */
+/*jshint white:false */
+/*jshint trailing:false */
+/*jshint newcap:false */
+
+var app = app || {};
+
+(function () {
+	'use strict';
+
+	var Utils = app.Utils;
+
+	app.todoActions = app.alt.createActions({
```

Comment:
most of this boilerplate could be reduced with [`generateActions`](https://github.com/goatslacker/alt/blob/master/docs/createActions.md#generateactions) as most of your actions are just passing the data through directly. The exception to this is `addTodo`, though I would probably create the default properties for the todo like `id` and `completed` in the store and just have `title` passed through via `generateActions`

something like:
```js
app.todoActions = app.alt.generateActions(
  'addTodo',
  'toggleAll',
  'toggle',
  'destroy',
  'save',
  'clearCompleted',
  'edit',
  'show'
);
```

for this change to work I believe you would also need to pass an object of data in the instances where you have more than one argument.